### DISCUSSION TITLE
2024/02/17/acm/codeforces/EducationalCodeforcesRound160/

### DISCUSSION THREAD

**Discussion by giscus** [Hukeqing/Hukeqing.github.io#14] [General(讨论贴)]
# 2024/02/17/acm/codeforces/EducationalCodeforcesRound160/

Educational Codeforces Round 160 (Rated for Div. 2) 个人写题记录

https://blog.mauve.icu/2024/02/17/acm/codeforces/EducationalCodeforcesRound160/

**Comment by kernelbin** [comment]
教教我

**Reply by Hukeqing** [reply]
不哦，应该是你教教我哦
